Cannot whitelist an RHSA vulnerability

If you are reporting a problem, please make sure the following information are provided:

**Expected behavior and actual behavior:**

I expect to be able to whitelist any detected vulnerability. However, while most images have a vulnerability starting with `CVE`, some have a vulnerability starting with `RHSA` (`rook/ceph:v1.0.5` for example). 

This behaviour is preventing me from using the feature which prevents vulnerable images from being pulled (I want to whitelist harmless vulnerbilities). For now, I'm forced to turn off vulnerability scanning on pull, since I can't pull images with RHSA vulnerabilities, even if they're harmless.

**Steps to reproduce the problem:**

Attempt to whitelist `RHSA-2019:2237` via the or the API. In the UI, the `Add` button is not clickable. Via the API, the error looks like this:

```
harbor_core.1.j2fv8msssaei@cn1    | 2019-09-24T11:40:29Z [ERROR] [/common/api/base.go:195]: failed to update project 1: invalid CVE ID: RHSA-2019:2099
harbor_core.1.j2fv8msssaei@cn1    | 2019-09-24T11:40:29Z [ERROR] [/common/api/base.go:228]:
```

**Versions:**
Please specify the versions of following systems.

- harbor version: 1.9.0
- docker engine version: `Docker version 19.03.2, build 6a30dfc`
- docker-compose version: N/A, using swarm

--- a/src/core/api/sys_cve_whitelist_test.go
+++ b/src/core/api/sys_cve_whitelist_test.go
@@ -115,6 +115,7 @@ func TestSysCVEWhitelistAPIPut(t *testing.T) {
 					ExpiresAt: &s,
 					Items: []models.CVEWhitelistItem{
 						{CVEID: "CVE-2019-12310"},
+						{CVEID: "RHSA-2019:2237"},
 					},
 				},
 				credential: sysAdmin,

--- a/src/pkg/scan/whitelist/validator.go
+++ b/src/pkg/scan/whitelist/validator.go
@@ -17,7 +17,6 @@ package whitelist
 import (
 	"fmt"
 	"github.com/goharbor/harbor/src/common/models"
-	"regexp"
 )
 
 type invalidErr struct {
@@ -46,11 +45,12 @@ const cveIDPattern = `^CVE-\d{4}-\d+$`
 // Validate help validates the CVE whitelist, to ensure the CVE ID is valid and there's no duplication
 func Validate(wl models.CVEWhitelist) error {
 	m := map[string]struct{}{}
-	re := regexp.MustCompile(cveIDPattern)
+	//	re := regexp.MustCompile(cveIDPattern)
 	for _, it := range wl.Items {
-		if !re.MatchString(it.CVEID) {
-			return &invalidErr{fmt.Sprintf("invalid CVE ID: %s", it.CVEID)}
-		}
+		//	 Bypass the cve format checking
+		//		if !re.MatchString(it.CVEID) {
+		//			return &invalidErr{fmt.Sprintf("invalid CVE ID: %s", it.CVEID)}
+		//		}
 		if _, ok := m[it.CVEID]; ok {
 			return &invalidErr{fmt.Sprintf("duplicate CVE ID in whitelist: %s", it.CVEID)}
 		}

--- a/src/pkg/scan/whitelist/validator_test.go
+++ b/src/pkg/scan/whitelist/validator_test.go
@@ -67,6 +67,7 @@ func TestValidate(t *testing.T) {
 			l: models.CVEWhitelist{
 				Items: []models.CVEWhitelistItem{
 					{CVEID: "breakit"},
+					{CVEID: "breakit"},
 				},
 			},
 			noError: false,

